docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/AggressiveShutdownStrategy.java

```diff
@@ -1,63 +0,0 @@
-/*
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
- */
-
-package com.palantir.docker.compose.execution;
-
-import static java.util.stream.Collectors.toList;
-
-import com.palantir.docker.compose.configuration.ShutdownStrategy;
-import com.palantir.docker.compose.connection.ContainerName;
-import java.io.IOException;
-import java.util.List;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-/**
- * Shuts down containers as fast as possible, without giving them time to finish
- * IO or clean up any resources.
- *
- * @deprecated Use {@link ShutdownStrategy#KILL_DOWN}
- */
-@Deprecated
-public class AggressiveShutdownStrategy implements ShutdownStrategy {
```

Comment:
Why are we removing deprecated classes in this PR?

Reply:
I had trouble getting the tests to pass for these shutdown strategies. Given that this PR changes the behavior of the shutdown strategies, it seemed like an appropriate time to remove them.